src/sql/azdata.d.ts

```diff
@@ -559,6 +559,8 @@ declare module 'azdata' {
 
 		disconnect(connectionUri: string): Thenable<boolean>;
 
+		changePassword(connectionUri: string, connectionInfo: ConnectionInfo, newPassword: string): Thenable<PasswordChangeResult>;
```

Comment:
All API updates should go in `azdata.proposed.d.ts`

And also have doc comments added.